apprentice_teacher.rb

```diff
@@ -1,21 +1,17 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'school'
+require_relative 'teachable'
+
+
+class ApprenticeTeacher < School
```

Comment:
An Apprentice Teacher is a (kind of) School?  That sounds a little off.  Make sure your naming matches conventions.